finality-provider/config/config.go

```diff
@@ -25,8 +25,8 @@ const (
 	defaultFinalityProviderKeyName = "finality-provider"
 	DefaultRPCPort                 = 12581
 	defaultConfigFileName          = "fpd.conf"
-	defaultNumPubRand              = 100
-	defaultNumPubRandMax           = 200
+	defaultNumPubRand              = 1000
+	defaultNumPubRandMax           = 1000
```

Comment:
Is this enough? I thought we need to commit pub rand that can last for 1-2 weeks

Reply:
You are right. We have calculated the number as 7k in https://github.com/babylonlabs-io/pm/blob/main/adr/adr-024-timestamping-public-randomness.md